lib/Elevate/Blockers.pm

```diff
@@ -121,7 +122,7 @@ sub _has_blockers ( $self, $check_mode = 0 ) {
     }
 
     $_CHECK_MODE = !!$check_mode;    # running with --check
-    $self->abort_on_first_blocker( !$_CHECK_MODE );
+    $self->abort_on_first_blocker(0);
```

Comment:
Was this testing to remove `!$_CHECK_MODE`?

Reply:
No.  For a while we had been always been calling _has_blockers with check mode enabled.  Check mode was always true and abort_on_first_blocker() always was initialized with 0.  The abort_on_first_blocker() would later be called with true for certain cases like when upcp was running.
I started calling this with check mode as "false" if called via --start.  I needed this so I would know whether to skip the leapp blocker.  But, I did not want to change the behavior with regards to blocking on the first blocker.